connect/governance/governance.py

```diff
@@ -427,24 +427,28 @@ def _print_model_changes_log(self):
             return
 
         model_info = self.get_model_info()
-        plan_model_info = self._get_model_info(plan_model)
-        match = True
         for key in model_info.keys():
             model_value = model_info[key]
             plan_model_value = plan_model[key]
             if model_value != plan_model_value:
-                match = False
-                global_logger.info(
-                    f"Platform model and local model {key} do not match. Platform {key}: {plan_model_value}, Local {key}: {model_value}"
-                )
-        if not match:
-            global_logger.info(
-                """
+                if key == "tags":
+                    global_logger.info(
+                        f"""
+        Platform model and local model tags do not match. Platform tags: {plan_model_value}, Local tags: {model_value}
         You can apply changes to governance by calling the following method:
             gov.apply_model_changes()
         Alternatively, calling gov.export() method will automatically apply changes to governance.
-                """
-            )
+                        """
+                    )
+                    
+                if key == "model_version":
+                    global_logger.info(
+                        f"""
+        Platform model and local model versions do not match. Platform version: {plan_model_value}, Local version: {model_value}
```

Comment:
If they don't update the version in the governance app, they can still send evidence right? It will just be tagged with the newest version and won't be associated with controls?

Reply:
You are right!